ui/app/mirrors/create/handlers.ts

```diff
@@ -33,6 +33,19 @@ export const IsQueuePeer = (peerType?: DBType): boolean => {
   );
 };
 
+const ValidSchemaQualifiedTarget = (
+  peerType: DBType,
+  tableName: string
+): boolean => {
+  const schemaRequiredPeer =
+    peerType === DBType.POSTGRES || peerType === DBType.SNOWFLAKE;
+  if (!schemaRequiredPeer) {
+    return true;
+  }
+
+  return tableName?.includes('.') && tableName?.split('.')[0].length !== 0;
```

Comment:
```suggestion
  return tableName && tableName.includes('.') && !tableName.startsWith('.');
```
With `?.` calling methods you also need to do `?.(args)` otherwise you end up with error about calling `null` as a function